### PR TITLE
Added table documenting the JWT claims for the OIDC cli command.

### PR DESCRIPTION
**Pull request by clbarrell**
Adding a table to document the JWT claims included in the OIDC cli command.

### CODE REVIEW

**Comment by clbarrell** [comment]
View it here: 

https://bk-docs-pr-2065.onrender.com/docs/agent/v3/cli-oidc

File: pages/agent/v3/cli_oidc.md [hunk 1]
```diff
@@ -4,21 +4,173 @@ The Buildkite Agent's `oidc request-token` command allows you to request an OIDC
...
+      <p>Subject</p>
+      <p>Identifies the subject of the JWT, typically representing the user or entity being authenticated.</p>
+      <p><em>Format:</em> <code>organization:ORGANIZATION_SLUG:pipeline:PIPELINE_SLUG:ref:REF:commit:BUILD_COMMIT:step:STEP_KEY</code>. If the build has a tag, <code>REF</code> is <code>refs/tags/TAG</code>. Otherwise, <code>REF</code> is <code>refs/heads/BRANCH</code>.</p>
+      <p><em>Example:</em> <code>organization:acme-inc:pipeline:super-duper-app:ref:refs/heads/main:commit:9f3182061f1e2cca4702c368cbc039b7dc9d4485:step:build</code></p>
+    </td>
+  </tr>
+   <tr>
+    <td><code>aud</code></td>
+    <td>
+      <p>Audience.</p>
```
Comment (mbelton-buildkite) [inline pages/agent/v3/cli_oidc.md:48]: Remove period for consistency:

```suggestion
      <p>Audience</p>
```

File: pages/agent/v3/cli_oidc.md [hunk 2]
```diff
@@ -4,21 +4,173 @@ The Buildkite Agent's `oidc request-token` command allows you to request an OIDC
...
+      <p>Identifies the intended audience for the JWT. Defaults to <code>https://buildkite.com/ORGANIZATION_SLUG</code> but can be overridden using the <code>--audience</code> flag</p>
+    </td>
+  </tr>
+   <tr>
+    <td><code>exp</code></td>
+    <td>
+      <p>Expiration Time</p>
```
Comment (mbelton-buildkite) [inline pages/agent/v3/cli_oidc.md:55]: Any particular reason for using title case here? If not, use sentence case:

```suggestion
      <p>Expiration time</p>
```

File: pages/agent/v3/cli_oidc.md [hunk 3]
```diff
@@ -4,21 +4,173 @@ The Buildkite Agent's `oidc request-token` command allows you to request an OIDC
...
+      <p>Specifies the expiration time of the JWT, after which the token is no longer valid. Defaults to 5 minutes in the future at generation, but can be overridden with the <code>--lifetime</code> flag.</p>
+      <p><em>Example:</em> <code>1669015898</code></p>
+    </td>
+  </tr>
+   <tr>
+    <td><code>nbf</code></td>
+    <td>
+      <p>Not Before</p>
```
Comment (mbelton-buildkite) [inline pages/agent/v3/cli_oidc.md:63]: Sentence case + the trailing space that's causing the linting to fail.

```suggestion
      <p>Not before</p>
```

File: pages/agent/v3/cli_oidc.md [hunk 4]
```diff
@@ -4,21 +4,173 @@ The Buildkite Agent's `oidc request-token` command allows you to request an OIDC
...
+    <td>
+      <p>Not Before</p> 
+      <p>Specifies the time before which the JWT must not be accepted for processing. Set to the current timestamp at generation.</p>
+      <p><em>Example:</em> <code>1669014898</code></p>
+    </td>
+  </tr>
+   <tr>
+    <td><code>iat</code></td>
+    <td>
+      <p>Issued At</p>
```
Comment (mbelton-buildkite) [inline pages/agent/v3/cli_oidc.md:71]: Sentence case:

```suggestion
      <p>Issued at</p>
```

File: pages/agent/v3/cli_oidc.md [hunk 5]
```diff
@@ -4,21 +4,173 @@ The Buildkite Agent's `oidc request-token` command allows you to request an OIDC
...
+    <td><code>build_commit</code></td>
+    <td>
+      <p>The SHA commit from the repository.</p>
+      <p><em>Example:</em> <code>9f3182061f1e2cca4702c368cbc039b7dc9d4485</code></p>
+    </td>
+  </tr>
+  <tr>
+    <td><code>step_key</code></td>
+    <td>
+      <p>The <code>key</code> attribute of the step from the pipeline. If the key is not set for the step, <code>nil</code> will be returned.</p>
```
Comment (mbelton-buildkite) [inline pages/agent/v3/cli_oidc.md:121]: Tightening:

```suggestion
      <p>The <code>key</code> attribute of the step from the pipeline. If the key is not set for the step, <code>nil</code> is returned.</p>
```

File: pages/agent/v3/cli_oidc.md [hunk 6]
```diff
@@ -4,21 +4,173 @@ The Buildkite Agent's `oidc request-token` command allows you to request an OIDC
...
+    <td>
+      <p>Audience.</p>
+      <p>Identifies the intended audience for the JWT. Defaults to <code>https://buildkite.com/ORGANIZATION_SLUG</code> but can be overridden using the <code>--audience</code> flag</p>
+    </td>
+  </tr>
+   <tr>
+    <td><code>exp</code></td>
+    <td>
+      <p>Expiration Time</p>
+      <p>Specifies the expiration time of the JWT, after which the token is no longer valid. Defaults to 5 minutes in the future at generation, but can be overridden with the <code>--lifetime</code> flag.</p>
```
Comment (mbelton-buildkite) [inline pages/agent/v3/cli_oidc.md:56]: Consistency:

```suggestion
      <p>Specifies the expiration time of the JWT, after which the token is no longer valid. Defaults to 5 minutes in the future at generation, but can be overridden using the <code>--lifetime</code> flag.</p>
```

**Review by mbelton-buildkite — APPROVED** [review]
Just a few nitpicks to address, then you're good to go :)

**Pull request merged**